input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
-                    <a
-                        href="https://build.fhir.org/ig/HL7/fhir-ipa/StructureDefinition-ipa-observation.html"
-                    >
-                        IPA-Observation
-                    </a>
-                </p>
-            </td>
-            <td width="111" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/>
-                </p>
-            </td>
-            <td width="68" valign="top">
-            </td>
-            <td width="130" valign="top">
-            </td>
-            <td width="78" valign="top">
-                <p>
-                    <a
-                        href="http://hl7.org/fhir/us/core/StructureDefinition/us-core-simple-observation"
-                    >
-                        US Core Simple Observation
-                    </a>
-                </p>
-            </td>
-            <td width="137" valign="top">
-                <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/>
-                </p>
-            </td>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-medicationrequest.html">AU Core MedicationRequest</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/uv/ipa/STU1/StructureDefinition-ipa-medicationrequest.html">IPA-MedicationRequest</a></td>
+            <td style="width: 25%;">MedicationRequest.requester</td>
+            <td style="width: 25%;">IPA prohibits reference to Organization and RelatedPerson resources.</td>
         </tr>
     </tbody>
-</table>
\ No newline at end of file
+</table>
+
+##### Missing Data
+AU Core and IPA share a compatible approach to handling missing data through the use of the DataAbsentReason extension.
+
+##### Suppressed Data
+AU Core provides additional guidance for handling data suppression due to security or privacy reasons, using 'unknown' or 'masked' codes from the DataAbsentReason Code System based on the requester's access rights.
+This ensures AU Core resources with suppressed data remain compliant with IPA through its use of the DataAbsentReason extension for missing data.
+
+##### Additional profiles
+This version of AU Core has no equivalent profile for the following IPA profiles:
+- IPA-DocumentReference
+- IPA-MedicationStatement
+
+#### International Patient Summary
+[International Patient Summary Implementation Guide 1.1.0](https://hl7.org/fhir/uv/ips/STU1.1/) describes how to represent the International Patient Summary (IPS) using HL7 FHIR. An International Patient Summary  document is an electronic health record extract containing essential healthcare information about a subject of care.
+
+AU Core is designed to be compatible with IPS, meaning AU Core data can be used to create a patient summary that aligns with IPS standards. To achieve compliance with both AU Core and IPS, additional requirements may need to be met. These requirements are outlined below.
```

Comment:
1. Replace "AU Core is designed to be compatible with IPS, meaning AU Core data can be used to create a patient summary that aligns with IPS standards. To achieve compliance with both AU Core and IPS, additional requirements may need to be met. These requirements are outlined below."
With "The following IPS profile(s) contain additional requirements. Implementers are advised to note that some code changes may be required to support these profiles."
2. Make equivalent change to US Core

Reply:
Done